src/services/post.ts

```diff
@@ -8,63 +11,249 @@ export const createPost = async (postInfo: IPostdetail): Promise<void> => {
     is_public: postInfo.is_public,
     user_id: postInfo.user_id,
     category_id: postInfo.category_id,
+    image_id: postInfo.image_id,
   });
 };
 
-export const getAllPost = async (): Promise<Post[] | null> => {
-  const allPost = await Post.findAll({
-    order: [['created_at', 'DESC']],
-  });
-  return allPost;
-};
-
-export const getPostPages = async (
-  page: number,
-  limit: number,
-): Promise<Post[]> => {
-  const offset = limit * (page - 1);
-  const pagePost = await Post.findAll({
-    offset,
-    limit,
-    order: [['created_at', 'DESC']],
-  });
-  return pagePost;
+export const getPost = async (
+  page: number | null,
+  limit: number | null,
+): Promise<Post[] | null> => {
+  if (!page && !limit) {
+    const allPost = await Post.findAll({
```

Comment:
page, limit이 주어지지 않았을 때의 기본값은 offset 0, limit 10으로 두는 것은 어떤가요?
코드도 줄어들고 !
```suggestion
export const getPost = async (
  page: number = 1,
  limit: number = 10,
): Promise<Post[] | null> => {
  const allPost = await Post.findAll({
```

Reply:
기본값 설정하는 방향으로 해보겠습니다!